doc-content/enterprise-only/openshift/environment-immutable-server-kjar-proc.adoc

```diff
@@ -0,0 +1,82 @@
+[id='environment-immutable-server-kjar-proc']
+// definitions for RH-SSO include
+:kieonly:
+// kieonly must be undefined at end of file
+:LOCAL_CENTRAL: {CENTRAL} Monitoring
+:modifylink: none
+= Deploying an immutable {KIE_SERVER} from KJAR services
+
+To deploy an immutable {KIE_SERVER} from KJAR services,
+ifdef::DM[]
+use the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-kieserver.yaml` template file. You can extract this file 
```

Comment:
yaml template file here, but simply yaml template in the previous section. Suggest you standarize. I like yaml template file.